.github/workflows/km-ci-workflow.yaml

```diff
@@ -456,6 +517,13 @@ jobs:
         kubectl apply -k cloud/k8s/deploy/kontain-deploy/ci
         sleep 20
 
+    # uncomment to get logs from damonset.
```

Comment:
Maybe `if env.SOMETHING == true` instead?

Reply:
I'll look into that. In the meantime, I'll push another commit with the debugging stuff removed.